lib/pbench/test/unit/server/query_apis/test_index_mappings.py

```diff
@@ -110,7 +108,7 @@ def test_with_no_index_document(self, client, server_config):
         present in the database.
         """
         with client:
-            response = client.get(f"{server_config.rest_uri}/index/mappings/run")
+            response = client.get(f"{server_config.rest_uri}/index/mappings/test")
```

Comment:
Since this is intended to fail, I would use a string which is more obviously bad than "test"...like "bad_data_object_type" or similar.